supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -23,8 +23,15 @@ Follow these guidelines when linking externally:
 * Always include link text. Bare URLs (URLs without explanatory information) are unhelpful because they do not provide adequate context about the link target.
 * Use hyperlinks unless the URL is an example URL or is otherwise inaccessible to users. For information about using addresses in examples, see the _IBM Style_ guide.
 * By default, links are followable and crawlable. Do not use the "nofollow" link option unless absolutely necessary.
-* Avoid links to external sites when possible. External links can change or be unreliable.
 * Avoid deep links (to a specific page or image from another company) to prevent an ongoing maintenance responsibility and to prevent bypassing a website's legal information.
```

Comment:
```suggestion
* Avoid deep links (to a specific page or image) to prevent an ongoing maintenance responsibility and to prevent bypassing a website's legal information.
```

Reply:
The original advice in IBM Style (about avoiding deep links) does not refer to external sites.